homeassistant/components/media_player/kodi.py

```diff
@@ -75,12 +86,17 @@
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
     vol.Optional(CONF_TCP_PORT, default=DEFAULT_TCP_PORT): cv.port,
     vol.Optional(CONF_PROXY_SSL, default=DEFAULT_PROXY_SSL): cv.boolean,
+    vol.Optional(CONF_TURN_ON_ACTION, default=None): vol.In(TURN_ON_ACTION),
```

Comment:
I've changed entity names for `cv.SCRIPT_SCHEMA` validation, but the symmetry lover that I am makes me keep the `turn_on_action` list, like the existent `turn_off_action` list, and ables to implement the `script_off` action as well, as one more option to the existent list, so no breaking change.
How do you see it now, @balloob?